test/users.test.js

```diff
@@ -0,0 +1,93 @@
+const request = require('supertest');
+const truncateDatabase = require('./setup');
+const app = require('../app');
+
+beforeAll(async () => {
+  process.env.NODE_ENV = 'test';
+  await truncateDatabase();
+});
+
+describe('POST users', () => {
+  const newUserData = {
```

Comment:
It's a good idea to move this mocked json to another file inside test folder, and then import it.

Reply:
Done :)